app/static/tests/e2e/code.etest.ts

```diff
@@ -86,11 +87,19 @@ test.describe("Code Tab tests", () => {
     });
 
     test("can reset code editor", async ({ page }) => {
-        const invalidCode = "deriv(y1) test * faker";
+        const invalidCode = "faker\n";
         await writeCode(page, invalidCode);
+        await expect(await page.locator(".run-tab .run-update-msg")).toHaveText(
+            "Model code has been updated. Compile code and Run Model to view updated graph.", {
+                timeout
+            }
+        );
         await expect(await page.innerText(".wodin-left .wodin-content #reset-btn")).toBe("Reset");
         await expect(await page.innerText(".wodin-left .wodin-content")).toContain("faker");
+        await expect(await page.innerText(".wodin-left .wodin-content #code-status")).toContain("Code is not valid");
         await page.click("#reset-btn");
+        await page.waitForResponse((response) => response.url().includes("/odin"));
         await expect(await page.innerText(".wodin-left .wodin-content")).not.toContain("faker");
+        await expect(await page.innerText(".wodin-left .wodin-content #code-status")).toContain("Code is valid");
```

Comment:
Great to get the status message check in - this comment from previous review hasn't been addressed though:
> it would also be good to test the default code beyond just checking that it's not identical to the fake code. Could actually save the value of the original default code on page load, and then check that the reset button reverts it to exactly that, to avoid making assumptions about exact code content.